packages/core/src/Cardano/types/StakePool/StakePool.ts

```diff
@@ -106,57 +95,55 @@ export class StakePoolEpochRewards {
 
   /**
    * The epoch number at which these rewards were calculated.
+   *
+   * @deprecated Use `epochNo` instead
    */
-  epoch: EpochNo;
+  epoch?: EpochNo;
 
-  /**
-   * Epoch length in milliseconds.
-   */
+  /** Epoch length in milliseconds. */
   epochLength: number;
 
-  /**
-   * The rewards generated by the pool that leaders have access to up to the current epoch.
-   * Incomplete for last two epochs.
-   */
+  /** The epoch number at which these rewards were calculated. */
+  epochNo: EpochNo;
+
+  /** The rewards generated by the pool that leaders have access to in the current epoch. */
   leaderRewards: Lovelace;
 
+  /** The part of the active stake delegated by members. */
+  memberActiveStake: Lovelace;
+
   /**
    * The Return Of Investment (ROI) from a delegator's perspective in the given epoch. Not annualized.
    *
    * Formula: memberRewards / (activeStake - pledge)
    *
    * Incomplete for last two epochs.
+   *
+   * @deprecated No longer supported
    */
-  memberROI: Percent;
+  memberROI?: Percent;
```

Comment:
This is a breaking change, please reflect in the commit message

Reply:
9bb6579a4 should solve this